ui/src/components/GridItem.vue

```diff
@@ -0,0 +1,99 @@
+<template lang="pug">
+  .item-row(ref="item-row")
+    .item-label(ref="item-label")
+      span(v-if="!$slots.label") {{ label }}
+
+      slot(name="label")
+
+    .item-value(
+      ref="item-value",
+      :locator="locator",
+    )
+      span(v-if="!$slots.value")
+        span(v-if="value") {{ value }}
+        span.text--disabled(v-else) —
+
+      slot(name="value")
+</template>
+
+<script>
+export default {
+  props: {
+    label: {
+      type: String,
+      required: false,
+    },
+
+    value: {
+      type: [String, Number],
+      required: false,
+    },
+
+    locator: {
```

Comment:
do we need it?

Reply:
No, I didn't clean GridItem. I can do it now or later, whatever you prefer.